src/promptflow-evals/promptflow/evals/evaluate/_evaluate.py

```diff
@@ -337,7 +337,7 @@ def evaluate(
     _validate_columns(input_data_df, evaluators, target, evaluator_config)
 
     pf_client = PFClient(
-        config={"trace.destination": trace_destination},
+        config={"trace.destination": trace_destination} if trace_destination else None,

```

Comment:
we might need to check trace_destination == "none" as well

when user unset trace destination through below command, trace_destination will return "none"
pf config set trace.destination=none

Reply:
Note: None is differnt from "none"

Reply:
I tried "None", "none" etc all seems to result in not tracking in cloud. So have made similar change